package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashgraph/solo",
-  "version": "0.24.0",
+  "version": "0.24.5",
```

Comment:
shouldn't this be auto-updating as part of our artifact build during tag/release?